pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>edu.illinois.cs</groupId>
     <artifactId>idflakies</artifactId>
-    <version>1.0.3-SNAPSHOT</version>
+    <version>1.1.3-SNAPSHOT</version>
```

Comment:
Let's update this instead to 1.1.0. We can jump up a minor version, but no need to keep that final 3.

Reply:
Done